build.bat

```diff
@@ -496,7 +560,7 @@ GOTO:EOF
 	echo.                           compilation.
 	echo.    --cs-build             Only C Sharp code generation and compilation.
 	echo.    --make  path           Path to the GNU make executable. If this
```

Comment:
Not applicable for this, but as a separate fix... I just realized that this make option does not make sense for Windows. We should remove it (this does not affect to this CR).

Reply:
I have filled a Github issue https://github.com/rticommunity/rtiperftest/issues/113